src/Mod/CleanStart/Gui/CleanStartView.cpp

```diff
@@ -0,0 +1,290 @@
+    iconLabel->setPixmap(baseIcon.pixmap(newFileIconSize, newFileIconSize));
+
+    auto textLayout = gsl::owner<QVBoxLayout*>(new QVBoxLayout);
+    auto textLabelLine1 = gsl::owner<QLabel*>(new QLabel(button));
+    textLabelLine1->setText(QLatin1String("<b>") + newButton.heading + QLatin1String("</b>"));
+    auto textLabelLine2 = gsl::owner<QLabel*>(new QLabel(button));
+    textLabelLine2->setText(newButton.description);
+    textLabelLine2->setWordWrap(true);
+    textLayout->addWidget(textLabelLine1);
+    textLayout->addWidget(textLabelLine2);
+    textLayout->setSpacing(0);
+    mainLayout->addItem(textLayout);
+
+    mainLayout->addStretch();
+
+    button->setMinimumHeight(newFileIconSize + cardSpacing);
+    return button;
+}
+
+}
+
+CleanStartView::CleanStartView(Gui::Document* pcDocument, QWidget* parent)
+    : Gui::MDIView(pcDocument, parent)
+    , _contents(new QScrollArea(parent))
+{
+    auto hGrp = App::GetApplication().GetParameterGroupByPath(
+        "User parameter:BaseApp/Preferences/Mod/Start");
+    auto cardSpacing = hGrp->GetInt("FileCardSpacing", 20);  // NOLINT
+
+    auto scrolledWidget = gsl::owner<QWidget*>(new QWidget(this));
+    _contents->setWidget(scrolledWidget);
+    _contents->setHorizontalScrollBarPolicy(Qt::ScrollBarPolicy::ScrollBarAlwaysOff);
+    _contents->setVerticalScrollBarPolicy(Qt::ScrollBarPolicy::ScrollBarAsNeeded);
+    _contents->setWidgetResizable(true);
+    auto layout = gsl::owner<QVBoxLayout*>(new QVBoxLayout(scrolledWidget));
+    layout->setSizeConstraint(QLayout::SizeConstraint::SetMinAndMaxSize);
+
+    auto newFileLabel = gsl::owner<QLabel*>(new QLabel(tr("New File")));
+    layout->addWidget(newFileLabel);
+    auto gridLayout = gsl::owner<QGridLayout*>(new QGridLayout);
+    layout->addLayout(gridLayout);
+    configureNewFileButtons(gridLayout);
+
+    auto recentFilesLabel = gsl::owner<QLabel*>(new QLabel(tr("Recent Files")));
+    layout->addWidget(recentFilesLabel);
+    auto recentFilesListWidget = gsl::owner<FileCardView*>(new FileCardView(_contents));
+    connect(recentFilesListWidget, &QListView::clicked, this, &CleanStartView::fileCardSelected);
+    layout->addWidget(recentFilesListWidget);
+
+    auto examplesLabel = gsl::owner<QLabel*>(new QLabel(tr("Examples")));
+    layout->addWidget(examplesLabel);
+    auto examplesListWidget = gsl::owner<FileCardView*>(new FileCardView(_contents));
+    connect(examplesListWidget, &QListView::clicked, this, &CleanStartView::fileCardSelected);
+    layout->addWidget(examplesListWidget);
+
+    layout->setSpacing(static_cast<int>(cardSpacing));
+    layout->addStretch();
+
+    setCentralWidget(_contents);
+
+    QString title = QCoreApplication::translate("Workbench", "Start");
+    setWindowTitle(title);
+
+    configureExamplesListWidget(examplesListWidget);
+    configureRecentFilesListWidget(recentFilesListWidget);
+}
+
+
+void CleanStartView::configureNewFileButtons(QGridLayout* layout)
```

Comment:
@chennes 
As the linter already reported this causes a build failure because the method is declared as const but in the source file it's not.